src/jats/__tests__/jats-exporter.test.ts

```diff
@@ -800,9 +800,8 @@ describe('JATS exporter', () => {
     projectBundle.data.find((el) => {
       if (el._id === 'MPEquation:C900DDA4-BE45-4AF6-8C9F-CA0AA5FCC403') {
         const equation = el as Equation
-        equation.MathMLStringRepresentation = mathMLFragment
+        equation.content = mathMLFragment
         // @ts-ignore
-        delete equation.TeXRepresentation
       }
```

Comment:
I'm guessing this if statement can be removed; it's not updating the correct field.